index.html

```diff
@@ -126,10 +126,22 @@
       </p>
       <section class="informative">
         <h3>
-          Example manifest
+          Example manifests
         </h3>
         <p>
-          The following shows a typical <a>manifest</a>.
+          The following shows a very simple <a>manifest</a>.
+        </p>
+        <pre class="example json" title="very simple manifest">
+{
+  "name": "Donate App",
+  "description": "This app helps you donate to worthy causes.",
+  "icons": [{
+    "src": "images/icon.png",
+    "sizes": "192x192"
+  }]
+}
+</pre>        <p>
```

Comment:
nit: please run over HTML5 tidy on the doc :) 

Also, you can align the example code with the pre tag if you wish. I think the highlighter takes care of alignment for you.